### DISCUSSION TITLE
【HEO主题】HEO_INFO_CARD_URL 带来的bug

### DISCUSSION THREAD

**Discussion by zqdongxw** [tangly1024/NotionNext#2020] [Q&A] [answered]
**描述bug**
【此项必填】HEO_INFO_CARD_URL 这个值留空或者注释掉，就会导致heo整个主题无法使用

**复现步骤**
【此项必填】HEO_INFO_CARD_URL 这个值留空或者注释掉

**期望的正常结果**
【此项必填】即使不填url，主题应该正常使用

**环境**

- 【必填】NotionNext版本 [例如. 4.3.0]
- 【必填】主题 [heo]
- 【必填】部署方案 [vercel]
- 【可选】操作系统: [windows]
- 【可选】浏览器 [edge]

**Comment by zqdongxw** [comment, marked as answer]
此字段保留且不留空，问题消失。

**Reply by tangly1024** [reply]
感谢反馈，已通过补丁避免此问题的出现https://github.com/tangly1024/NotionNext/commit/5b606213cb1c846a0277e6c2e6864d8d9816a836
